src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -26,12 +27,15 @@ public class ActorRuntime
         // Map of ActorType --> ActorManager.
         private readonly Dictionary<string, ActorManager> actorManagers = new Dictionary<string, ActorManager>();
 
+        private ActorSettings actorSettings;
+
         /// <remarks>
         /// WARNING: This type is expected to be accessed via the <see cref="Instance" /> singleton instance.
         /// This constructor is exposed only for unit testing purposes.
         /// </remarks>
         internal ActorRuntime()
         {
+            this.actorSettings = new ActorSettings();
```

Comment:
Ideally this should be created, only when user hasn't provided one with UseActorSettings. If we are creating a default instance then exposing it through ConfigureACtorSettings as discussed [here](https://github.com/dapr/dotnet-sdk/pull/269#discussion_r399728003) would make more sense